livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/llm.py

```diff
@@ -696,6 +700,15 @@ async def _run(self) -> None:
             opts = _strip_nones(opts)
 
             messages = _build_oai_context(self._chat_ctx, id(self))
+            if (
+                self._llm._capabilities.requires_initial_user_message
+                and len(messages) > 0
+            ):
+                if messages[0]["role"] == "system" and messages[1]["role"] != "user":
+                    messages.insert(1, {"role": "user", "content": ""})
+                elif messages[0]["role"] != "user":
```

Comment:
I'm not sure if this is sufficient.. I think Perplexity (and others) require interleaved user / assistant turns.. 

IMO we need to scan through the messages and enforce interleaved messages. maybe also renaming the capability: `single_message_per_turn`?